File: apps/provider-api/Models/VwCurrentInstitutionLocationWebsite.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitutionLocationWebsite
    {
        public int? InstitutionLocationWebsiteId { get; set; }
        public int InstitutionId { get; set; }
        public int InstitutionLocationId { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionCode { get; set; }
        public string InstitutionStatus { get; set; }
        public string LocationName { get; set; }
        public int? LocationNumber { get; set; }
        public string LocationStatus { get; set; }
        public string WebsiteUsage { get; set; }
        public int? ListSequence { get; set; }
        public string WebsiteUrl { get; set; }
        public Guid? WebsiteGuid { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/vw-current-institution-location-website.ts



export class VwCurrentInstitutionLocationWebsite {
    public institutionLocationWebsiteId: number;
    public institutionId: number = 0;
    public institutionLocationId: number = 0;
    public institutionName: string = '';
    public institutionCode: string = '';
    public institutionStatus: string = '';
    public locationName: string = '';
    public locationNumber: number;
    public locationStatus: string = '';
    public websiteUsage: string = '';
    public listSequence: number;
    public websiteUrl: string = '';
    public websiteGuid: string;

    constructor(init? : Partial<VwCurrentInstitutionLocationWebsite>) {
        
        Object.assign(this, init);
    }
}

